operator_test.go

```diff
@@ -1276,3 +1276,35 @@ func newRunNResult(t *testing.T, total int64, results []*RunResult) *runNResult
 	r.RunResults = results
 	return r
 }
+
+func TestRunUsingHTTPOpenAPI3(t *testing.T) {
+	tests := []struct {
+		skipValidateRequest string
+		wantErr             bool
+	}{
+		{"false", true},
+		{"true", false},
+	}
+	for _, tt := range tests {
+		t.Run(fmt.Sprintf("skipValidateRequest: %s", tt.skipValidateRequest), func(t *testing.T) {
+			ctx := context.Background()
+			ts := testutil.HTTPServer(t)
+			t.Setenv("TEST_HTTP_END_POINT", ts.URL)
+			t.Setenv("SKIP", tt.skipValidateRequest)
```

Comment:
```suggestion
			t.Setenv("TEST_SKIP_VALIDATE_REQUEST", tt.skipValidateRequest)
```

It is easy to see from the environment variable names which fields are reflected.